src/acom_music_box/music_box_model_options.py

```diff
@@ -12,7 +12,7 @@ class BoxModelOptions:
         simulationLength (float): Length of the simulation in hours.
     """
 
-    def __init__(self, chem_step_time, output_step_time, simulation_length, grid="box"):
+    def __init__(self, chem_step_time=1.0, output_step_time=5.0, simulation_length=100.0, grid="box"):
```

Comment:
same comment as above

Reply:
**None** works and produces sensible floating output.